scripts/avail-ext-doc/README.md

```diff
@@ -2,16 +2,34 @@
 
 Automatically create PR to update the document listing the available CLI extensions.
 
+To build locally:
+
 ```bash
-docker build . -t derekbekoe/az-avail-ext-doc-sync:0.1.2
+docker build . -t sttramer/az-ext-list-publisher:0.2.0
 ```
 
 To run locally:
+
 ```bash
-docker run --rm -e TRAVIS_BUILD_ID=$TRAVIS_BUILD_ID -e TRAVIS_REPO_SLUG=$TRAVIS_REPO_SLUG -e TRAVIS_COMMIT=$TRAVIS_COMMIT -e GH_TOKEN=$GH_TOKEN -e DOC_REPO_SLUG=$DOC_REPO_SLUG -v $PWD:/repo derekbekoe/az-avail-ext-doc-sync:0.1.2
+docker run --rm -e TRAVIS_BUILD_ID=$TRAVIS_BUILD_ID \
```

Comment:
If we decide to split the line for each parameter then break the line at `--rm` and `-e` here as well.